<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>        Функция</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="funcBox">
           <item>
            <property name="text">
             <string>(x^2+y-11)^2 + (x+y^2-7)^2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x^2 + 3*y^2 + 2*x*y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>-12*y + 4*x^2 + 4*y^2 - 4*x*y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>(x-2)^4 + (x-2*y)^2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8*x^2 + 4*x*y + 5*y</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x^3 + y^2 - 3*y - 2*x + 2</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>       Метод кодирования</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="codeBox">
           <item>
            <property name="text">
             <string>integer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>real</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>       Диапазон</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineXMIN"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineXMAX"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>       Точность</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineAccuracy"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>   Коэффициент мутации</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineMutationK"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>       a1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineA1"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>       a2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineA2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>       Количество частиц</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineN"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="initButton">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algoBox">
       <item>
        <property name="text">
         <string>Genetic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Swarm</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
